main.go

```diff
@@ -91,25 +97,26 @@ func newSteveServer(c stevecli.Config, ctx context.Context) (*steveserver.Server
 		return nil, err
 	}
 
-	client, err := initKubeClient(c.KubeConfig)
+	client, dyclient, err := initKubeClient(c.KubeConfig)
 	if err != nil {
 		return nil, err
 	}
 
 	restConfig.RateLimiter = ratelimit.None
 
-	a := auth.NewK3sAuthenticator(restConfig.Host, client, ctx)
+	//a := auth.NewK3sAuthenticator(restConfig.Host, client, ctx)
 	edgeServer := &edgeserver.EdgeServer{
 		RestConfig: restConfig,
 		Client:     client,
+		DyClient:   dyclient,
 		Context:    ctx,
 	}
 
 	handler := router.New(edgeServer)
 
 	return &steveserver.Server{
-		RestConfig:     restConfig,
-		AuthMiddleware: auth.ToAuthMiddleware(a),
+		RestConfig: restConfig,
+		//AuthMiddleware: auth.ToAuthMiddleware(a),
```

Comment:
please double-check your PR of `file changes` after submission, and don't leave comment out code.